packages/rocketchat-language-plugin/server/get_language.js

```diff
@@ -0,0 +1,13 @@
+Meteor.methods({
+	get_language() {
```

Comment:
If this is a method called from server-side only, then you do not have to expose is as Meteor method, but simply as a JavaScript function you call.